.github/workflows/unit-test.yml

```diff
@@ -45,7 +45,10 @@ jobs:
     - name: Test with pytest
       run: |
         # Test everything except the flink tests
-        pytest -vvv -s --cov=pangeo_forge_runner --ignore=tests/test_flink.py tests/
+        pytest -vvv -s --cov=pangeo_forge_runner \
```

Comment:
Let's split the tests into a 'unit/' directory and an 'integration/' directory, so this can just run unit tests instead of having to explicitly add each integration test as an --ignore?

Reply:
Done in [a6da8c0](https://github.com/pangeo-forge/pangeo-forge-runner/pull/52/commits/a6da8c061f9a8dad4caf55f0b9a4547fb3e93799)